### ISSUE TITLE
[macOS] Slow perf/repaint when resizing columns in Avalonia 11

### ISSUE THREAD

**Issue by danipen** [AvaloniaUI/Avalonia.Controls.TreeDataGrid#159]
Hi, this issue happens only in macOS.

We experience slow performance (or probably that the control is not repainted) when moving the columns header splitter in the TreeDataGrid.

https://user-images.githubusercontent.com/501613/228465740-a28c0961-425c-432b-9564-e96972c651d3.mov

@grokys @wieslawsoltes @jmacato can you reproduce it too? I'm just trying latest changes on master.

**Comment by danipen** [comment]
Note that under windows, the resize is normal:
![resize](https://user-images.githubusercontent.com/501613/228466707-6ebf70bd-e6e8-4f17-9627-1ba45abcbb4e.gif)

**Comment by grokys** [comment]
No, that PR is unrelated I'm afraid. I'm no closer to understanding why macOS is slowing down when using an opaque background, unfortunately.
